app/utils/clone-learner-group.js

```diff
@@ -0,0 +1,32 @@
+import RSVP from 'rsvp';
+
+const { map } = RSVP;
+
+export default async function cloneLearnerGroup(store, group, cohort, withLearners, parent = null) {
```

Comment:
please add a docblock here, outlining structure of returned array (first item is copy of of given group etc).